### DISCUSSION TITLE
webMAN MOD 1.47.44

### DISCUSSION THREAD

**Discussion by aldostools** [aldostools/webMAN-MOD#836] [General]
⭐ Added detection of the newest Evilnat 4.90.1 PEX (DEX mode)
🗑️ Uninstall webMAN now also deletes /dev_hdd0/tmp/gameboot folder
☑️ PS2 config database updates with fixes by @mrjaredbeta, @agrippa, @kozarovv
🚀 Cumulative update (includes over 19 commits to the suite of tools after 1.47.43)

<hr>
💡 <b>If you like webMAN MOD, then feel free to contribute with a *small donation</font>*, I'd really appreciate it!<BR>
&nbsp;  &nbsp; &nbsp; <a href="https://www.paypal.me/aldostools">paypal.me/aldostools</a> or <a href="https://donate.aldostools.org">donate.aldostools.org</a> (Funds are mainly to cover the high server costs)</b><br>
<br>
Download: <a href="https://github.com/aldostools/Resources/releases/download/1.0/EP0001-BLES80608_00-COVERS0000000000.pkg">Covers Pack [PS3]</a> &bull; <a href="https://github.com/aldostools/Resources/releases/download/1.0/EP0001-BLES80608_00-COVERS00000RETRO.pkg">Covers Pack [PSX/PS2]</a> &bull; <a href="https://www.mediafire.com/file/rrkpa28wtxk6btd">Retro Arch CE</a> &bull; <a href="https://github.com/aldostools/Resources/releases/tag/Addons">Addons</a> &bull; <a href="https://github.com/aldostools/Resources/releases/tag/Themes">Themes</a>  &bull; <a href="https://github.com/aldostools/webMAN-MOD/wiki">Documentation</a> &bull; <a href="https://github.com/aldostools/resources">More</a>

<hr /><em>This discussion was created from the release <a href='https://github.com/aldostools/webMAN-MOD/releases/tag/1.47.44'>webMAN MOD 1.47.44</a>.</em>

**Comment by millsey1980** [comment]
Hi i love your work for the ps3 scene i have only been on the scene for around two years but i have seen great things about you.
But i do have a question i think only you may be able to answer i have a friend who is on hen 4.90 and since installing wbman he cannot open any homebrew other than webman, he has tried to disable the admin setting in webman but he is still not able to launch any homebrew would you possibly know the problem 
thank you

**Reply by aldostools** [reply]
> Hi i love your work for the ps3 scene i have only been on the scene for around two years but i have seen great things about you. But i do have a question i think only you may be able to answer i have a friend who is on hen 4.90 and since installing wbman he cannot open any homebrew other than webman, he has tried to disable the admin setting in webman but he is still not able to launch any homebrew would you possibly know the problem thank you

I recommend that uninstall webMAN MOD and check again.
If the problem persists, the cause is something else like another homebrew or configuration.

If the problem is fixed, the issue could be incorrect settings in webMAN or was using an old version.

webMAN MOD must be updated to version 1.47.44 or later to use it on HFW 4.90.1 + HEN or CFW 4.90.

**Reply by millsey1980** [reply]
thank you i will get him to try that and see how he gets on

**Comment by AlenoxNoEnd** [comment]
I have a ps3 with damaged wifi and bluetooth, and because of that I haven't been able to use its webman for years, any version, a notification always appears that the webman has not been loaded, the webman configuration page does not load and give a error numbering.  so please could you somehow get the webman to work on a ps3 with broken wifi/bluetooth.  and I am grateful for your work

**Reply by aldostools** [reply]
> I have a ps3 with damaged wifi and bluetooth, and because of that I haven't been able to use its webman for years, any version, a notification always appears that the webman has not been loaded, the webman configuration page does not load and give a error numbering. so please could you somehow get the webman to work on a ps3 with broken wifi/bluetooth. and I am grateful for your work

If your console is CFW compatible, I recommend that install Evilnat CFW 4.90 PEX noBT.
https://mega.nz/folder/oV93nACL#MGK4ajtgpT5EBrBL8yjvRg/folder/UJ1ERZ7B

If your console is not CFW compatible (e.g. slim 25xx, 30xx or superslim), your options are: replace the wifi+bluetooth module or buy another console CFW compatible. e.g. all phat, slim 20xx & 21xx models are 100% compatible. Only certain models of slim 25xx are CFW compatible, you would need to use minchkver to know.

**Reply by AlenoxNoEnd** [reply]
thanks for answering. so i installed cfw 4.89.3 noBD/noBT CEX evilnat and went to test webman (i think it was version 1.47.43) but it didn't work, it's like my ps3 doesn't load the webman plugin, whenever i try to run something function or when I try to start a game, it gives a notification: "webman not load", and also it does not load the settings page in the browser, instead it gives an error and a number that I will not remember now, for now I can't install this new evilnat cfw because my ps3 has a power problem, but as soon as I fix that, I went to test it and let you know if it worked

**Comment by aldostools** [comment]
I don't have a console with noBT to test, but I guess that notification is because the network is disabled or not recognized.

I'd suggest you ask in PSX-PLACE forums.

**Reply by AlenoxNoEnd** [reply]
I'm sorry, actually the notification is: "webMAN not ready". but it doesn't work even i connect the ethernet cable. I know there was a cfw 4.85 noBT OVERFLOW that had the modified webman on it, I can't say which version it was but it worked.  but thanks for answering me

**Comment by ImperialFist40K** [comment]
Hello, I've always used HEN, but I decided to switch to CFW yesterday. Everything went fine until Webman couldn't be installed on my console (PS3 Slim). I kept trying and finally managed to install both Webman and multiman. But now I keep having trouble. I'm trying to install other PKGs but the console shows me these errors: "Not enough space in the internal storage" and "Error: 8001003A". I still have 15 GB of free space and I always activate dev_blind to install those other PKGs. Do you know what's causing those issues and how to fix them? Maybe I have to install a different version of CFW, because the fan sometimes changes speed randomly and it's not because overheating or something. My CFW is the standard Evilnat 4.90 Cobra 8.4 [PEX] [CEX]. Thanks.

**Reply by aldostools** [reply]
Did you install webMAN MOD 1.47.44 ??? Older versions don't work or cause bugs due they don't support 4.90

**Comment by MichaelX17** [comment]
> ⭐ Added detection of the newest Evilnat 4.90.1 PEX (DEX mode) 🗑️ Uninstall webMAN now also deletes /dev_hdd0/tmp/gameboot folder ☑️ PS2 config database updates with fixes by @mrjaredbeta, @agrippa, @kozarovv 🚀 Cumulative update (includes over 19 commits to the suite of tools after 1.47.43)
> 
> 💡 If you like webMAN MOD, then feel free to contribute with a *small donation*, I'd really appreciate it!
>       [paypal.me/aldostools](https://www.paypal.me/aldostools) or [donate.aldostools.org](https://donate.aldostools.org) (Funds are mainly to cover the high server costs)
> 
> Download: [Covers Pack [PS3]](https://github.com/aldostools/Resources/releases/download/1.0/EP0001-BLES80608_00-COVERS0000000000.pkg) • [Covers Pack [PSX/PS2]](https://github.com/aldostools/Resources/releases/download/1.0/EP0001-BLES80608_00-COVERS00000RETRO.pkg) • [Retro Arch CE](https://www.mediafire.com/file/rrkpa28wtxk6btd) • [Addons](https://github.com/aldostools/Resources/releases/tag/Addons) • [Themes](https://github.com/aldostools/Resources/releases/tag/Themes) • [Documentation](https://github.com/aldostools/webMAN-MOD/wiki) • [More](https://github.com/aldostools/resources)
> _This discussion was created from the release [webMAN MOD 1.47.44](https://github.com/aldostools/webMAN-MOD/releases/tag/1.47.44)._

Hi, man, i really appreciate your work, i've been using WebMan since 2019 and it really works very well, best PS3 Hombrew ever, but i would like to now something. Why did you remove the 'Refresh XML' option? That was a very helpful option to avoid rebooting our consoles after adding some ISO/Folder games to our library

**Comment by aldostools** [comment]
I didn't remove the option, at least intentionally. 
I will check again in case the was some typo in the xml

You can use select+L3 to refresh at any moment

**Reply by MichaelX17** [reply]
Thanks for your help, man, i works

**Comment by fabiomontteiro** [comment]
Hi man, I really appreciate your work. Just for feedback, I also have a ps3 with bluray and bluetooth issues. I use CFW 4.90[PEX][CEX][noBD+noBT] and I cannot access any option of webmanmod, always the message "webman is not ready yet". When accessing any configuration that opens the browser, error 8071053d appears.

**Comment by aldostools** [comment]
Maybe this tip is unrelated, but I will post it anyway in case it could be helpful for someone:

* If your PS3 has a CFW noBD (in any of its variants) and latest version of webMAN MOD, there are chances
 that the PS3 can’t read an external devices (/dev_usb000, /dev_usb001, …) in the next two minutes.
 To fix this, go to webMAN Setup page, click on webMAN MOD Setup and check noBD patch and reboot the PS3,
 now it may read external devices

**Comment by gau3191** [comment]
oi, primeiramente agradeço todo seu esforço e compartilhamento de informações! mas gostaria de esclarecer uma dúvida eu tenho jm ps3 desbloqueado cfw e atualizei pra versao pex(dex) 4.90 e qdo tentei entrar na psn nao estava habilitando, tentei r2+/\ pelas configurações, ai eu atualizei o webman pra essa ultima versão 1.47.44 e continuou o problema, resolvi desinstalar e ate deu pra iniciar seção psn, mas nao conclui, assisti alguns videos de como fazer a desinstalação do webman e diz q deve terminar de excluir ele pelo multiman mas as informações das pastas q estão nos vídeos são boot_pluguins.txt e a que esta aparecndo pra mim no muntilman é boot_pluguins_nocobra.txt sabe dizer se éa mesma coisa?  é so essa mesmo q deve ser removida pra deletar tudo do webMan?

**Comment by aldostools** [comment]
Delete the file `/dev_hdd0/tmp/wm_res/npsignin_plugin.rco`

This rco is used by webMAN MOD to block the PSN access when the CFW syscalls are enabled. 
webMAN MOD 1.47.44 not longer copies the file to that folder, but if the file exists from a previous installation it will be used.

Cobra 8.4 in Evilnat 4.90 has a similar feature. To disable it, it requires to delete `/dev_blind/vsh/resource/npsignin_plugin.lck`

**Reply by gau3191** [reply]
muito obrigada amigo pela resposta, mas só pra ter certeza é pra excluir todos esses arquivos que você indicou ou pra entrar de pasta em pasta e excluir somente o ultimo que vc escreveu? e no caso eu ja exclui o webmn daria pra eu fazer essa exclusão pelo multiman?

**Reply by Zoukambada** [reply]
Thanks Aldo.

I had a PSN error code 8071053D everytime I tried to login. I deleted the npsignin_plugin.rco file and it started to work again!

If anyone is interested, here's the PSX-Place thread that a reported my PSN password / login /connection problems:

https://www.psx-place.com/threads/problem-logging-into-psn-with-qrcode.42849/#post-395609

**Comment by aldostools** [comment]
You can delete both files safely. They are not used directly. They are only remapped to the rco.
`/dev_hdd0/tmp/wm_res/npsignin_plugin.rco`
`/dev_blind/vsh/resource/npsignin_plugin.lck`

You have to reboot after delete them without try to access to PSN.

If you try to access to PSN with the deleted files still mapped to the system file in dev_flash, the system will crash and you may need to reinstall the CFW again.

**Reply by gau3191** [reply]
vlw bro👊🏽

**Comment by Dannydai123** [comment]
update to .44 , webman mod folder was empty. not working any more. not able to uninstall it

**Reply by aldostools** [reply]
The empty folder usually happens when the plugin is still scanning the content of your system.

You can try unplugging all USB devices and rebooting to speedup the content scanning.

When the options appears go to /setup.ps3, check the option Disable content scan on startup, save the settings and reboot.
After this change you can use prepISO to scan the new content or SELECT+L3.

**Comment by Zenti11** [comment]
Hey aldo I'm having an issue with 1.47.44 full version. I'm on Evilnat D-PEX 4.90 and each time I load a game; iso or HDD, it will freeze the console. Thanks for helping.

**Reply by aldostools** [reply]
I recommend you that change Cobra to debug edition using CFW Tools menu, then use socat to capture the cobra's log.

> Create a new folder on the desktop (or any other path of your preference) and run it from there.
>I use this [socat-1.7.3.2-1-x86_64.zip](https://github.com/aldostools/webMAN-MOD/files/11266425/socat-1.7.3.2-1-x86_64.zip)
>Run socat.bat

The log should give you a clue about which is the last file or process accessed.

**Reply by Zenti11** [reply]
Awesome, thanks.

**Comment by holera91** [comment]
Hi!You did great work! I have a issue: when i try to connect with ftp on my mac or windows all files is read only, i cant copy any files to my console. Ps3 SS,4.90 webman .44

**Reply by aldostools** [reply]
Theh FTP server works with a variety of FTP clients, but there are limitations with some of them.

The recommended FTP client is FileZilla
https://filezilla-project.org/download.php

**Comment by VivaKalman666** [comment]
[Covers Pack [PS3], problem it's game 007: Quantum of Solace has a disk code ([BLES-00406 ], [BLES-00407 ]) .
Game sega mega drive ultimate collection has a disk code  [BLES-00407 ].I would really appreciate if you fix this for all users.

**Reply by aldostools** [reply]
Thanks for your feedback. I have fixed this cover in the pack.
https://github.com/aldostools/Resources/releases/tag/1.0

**Comment by palacios23910** [comment]
Hola alguien me ayuda porq en mi PS3 no puedo jugar los juegos a lo q inicio el juego me sale una pantalla negra ayuda por favor
